views/authed.twig

```diff
@@ -241,10 +241,16 @@
             <td>
                 <span class="preview"></span>
             </td>
-            <td class="title">
+            <td class="title" title="Name">
```

Comment:
Translations are probably needed here, somehow...

Reply:
I'm not sure how to do that there 🤔